default/python/AI/ShipDesignAI.py

```diff
@@ -1751,28 +1911,10 @@ def __init__(self):
     def _rating_function(self):
         if self.fighter_capacity < 1:
             return INVALID_DESIGN_RATING
-        # first, calculate "normal" weapon stuff
-        weapon_dmg = max(self._total_dmg_vs_shields(), self._total_dmg() / 1000)
-        effective_structure = self._effective_structure()
-
-        # now, consider offensive potential of our fighters
```

Comment:
Looks like this part totally dropped, I assume it is code duplication you talk about?